kvbc/include/merkle_tree_db_adapter.h

```diff
@@ -15,120 +16,117 @@
 #include <memory>
 #include "storage/db_types.h"
 
-namespace concord {
-namespace storage {
-namespace blockchain {
-namespace v2MerkleTree {
+namespace concord::kvbc::v2MerkleTree {
 
 class DBKeyManipulator {
  public:
-  static concordUtils::Sliver genBlockDbKey(kvbc::BlockId version);
+  static concordUtils::Sliver genBlockDbKey(BlockId version);
```

Comment:
These gen...() functions should return Key

Reply:
Good point, done.